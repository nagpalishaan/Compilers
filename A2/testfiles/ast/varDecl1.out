Program
  List
    Function
      IdDecl ID="n"
      Opt
      List
        DeclareVar
          IdDecl ID="x"
          Opt
            Numeral NUMERAL="3"
        DeclareVar
          IdDecl ID="y"
          Opt
            Sub
              IdUse ID="x"
              Numeral NUMERAL="2"
