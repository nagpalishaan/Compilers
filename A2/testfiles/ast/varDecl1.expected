Program
  List
    Function
      IdDecl ID="n"
      Opt
        List
          DeclareVar
            IdDecl ID="x"
            List
              Numeral NUMERAL="3"
          DeclareVar
            IdDecl ID="y"
            List
              Sub
                IdUse ID="x"
                Numeral NUMERAL="2"
